src/components/calculator/PointsDisplay.tsx

```diff
@@ -78,77 +82,150 @@ export function ExplanationCard(props: {
+    (points / weekBudget) * 100,
+  )}% of your weekly risk budget`
 }
 
 export function PointsDisplay(props: {
   points: number
   repeatedEvent: boolean
+  riskBudget: number
+  riskBudgetSetter: (newValue: number) => void
   upperBound: number
   lowerBound: number
 }): React.ReactElement {
+  const currentRiskLevel = howRisky(props.points, props.riskBudget)
+  const doShowPoints = showPoints(props.points)
   return (
-    <div className="top-half-card">
-      <strong>Results:</strong>
-      {showPoints(props.points) ? (
-        <h1>
-          {displayPoints(props.points)} microCOVIDs (
-          {displayPoints(props.lowerBound)} to {displayPoints(props.upperBound)}
-          ){props.repeatedEvent ? ' per week' : ' each time'}
-        </h1>
-      ) : (
-        <h1>fill in calculator to see</h1>
-      )}
-    </div>
+    <Row className="top-half-card no-gutters">
+      <Col className="legend-container">
+        {doShowPoints && currentRiskLevel.icon ? (
+          <>
+            <currentRiskLevel.icon
+              className={'risk-icon text-risk-' + currentRiskLevel.style}
+            />
+          </>
+        ) : (
+          riskLevels
+            .map((level) => level.style)
+            .slice(0, RISK_LEVELS_TO_SHOW_ON_LEGEND) // Makes a shallow copy of the displayable risk levels
+            .reverse() // Reverse the sort so the lowest items are at the bottom
+            .map((level) => (
+              <div
+                key={level}
+                className={
+                  `legend-piece risk-${level}` +
+                  (!doShowPoints || level !== currentRiskLevel.style
+                    ? ''
+                    : ' current-level')
+                }
+              ></div>
+            ))
+        )}
+      </Col>
+      <Col md="11" sm="10" xs="10" className="points-container">
+        {!doShowPoints ? (
+          <div className="risk-level"></div>
+        ) : (
+          <div className={'risk-level risk-' + currentRiskLevel.style}>
+            <span>{currentRiskLevel.title} Risk</span>
+          </div>
+        )}
+        <div className="budget-consumption">
+          {doShowPoints && (
+            <>
+              {budgetConsumption(props.points, props.riskBudget)}{' '}
+              <span className="points-range d-md-inline d-none">
+                (of {displayPoints(getWeekBudget(props.riskBudget))}{' '}
+                microCOVIDs/week)
+              </span>
+            </>
+          )}
+        </div>
+        <div className="points">
+          {doShowPoints ? (
+            <>
+              {tooManyPoints(props.points) ? '>' : '~'}
+              {displayPoints(props.points)} microCOVIDs
+              {props.repeatedEvent ? ' per week' : ' each time'}{' '}
+              <span className="points-range d-md-inline d-none">
+                {props.upperBound >= ONE_MILLION ? null : (
+                  <>
+                    (range: {displayPoints(props.lowerBound)} to{' '}
```

Comment:
Could we say "90% interval" instead of "range"? 

I worry that most people don't understand how to parse "range" (at best assume that these are 100% confidence intervals?) and people who understand stats also don't know how to parse "range" (is it standard deviation? 90% p<.05?)

Reply:
Moved to Slack convo.